### ISSUE TITLE
[Accessibility] "Ensure elements with ARIA role that require child roles contain them

### ISSUE THREAD

**Issue by vasanthkumar258** [jquense/react-big-calendar#2538]
### Check that this is really a bug

- [X] I confirm

### Reproduction link

https://jquense.github.io/react-big-calendar/examples/index.html?path=/story/about-big-calendar--page

### Bug description

when we are having a show more button..

Issue - div[role="row"] is having button[tabindex]
![image](https://github.com/jquense/react-big-calendar/assets/12809041/5374e674-6be4-4381-88d7-4b5f53ca6694)

Possible fix - We need to add role="cell" to the parent div of the "Show more" button
![image](https://github.com/jquense/react-big-calendar/assets/12809041/0cbaca77-b136-415a-91b3-f8c9f2cef001)

### Expected Behavior

No Accessibility issue should be found

### Actual Behavior

Accessibility issue is observed.

### react-big-calendar version

1.11.1

### React version

Recat v16.8

### Platform/Target and Browser Versions

Edge

### Validations

- [X] Read the [docs](http://jquense.github.io/react-big-calendar/examples).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.
- [X] Make sure this is a react-big-calendar issue and not an implementation issue

### Would you like to open a PR for this bug?

- [ ] I'm willing to open a PR

**Comment by vasanthkumar258** [comment]
@cutterbl, any update on this?/

**Comment by carolineWallop** [comment]
Wondering if we can add a fix for this? I'm encountering the same accessibility error, and can confirm that adding role="cell" fixes it.

**Comment by rasmus-davidsson** [comment]
Would also like this to be fixed :)
